oidc-handler/ui/src/main/resources/SLING-INF/content/login/login.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
```

Comment:
This seems to have lots of extra classes which are referenced but not added to the page. I suggest that you start with a minimal page with no extra markup, we can take care of styling it later.